enhancements/network/egress-qos.md

```diff
@@ -0,0 +1,119 @@
+---
+title: OVN Pods Egress DSCP QoS
+authors:
+  - "@oribon"
+reviewers:
+  - "@trozet"
+  - "@danwinship"
+  - "@tssurya"
+approvers:
+  - TBD
```

Comment:
suggestion: you can make Tim approver for OVN-K side, Dan for API side, Ben Bennett who leads networking.